utils/markdown.js

```diff
@@ -141,10 +141,12 @@ function buildTree(lines) {
     }
 
     if (content.startsWith(':::')) {
+      let noteType = content.substring(3).split(' ');
```

Comment:
Can we keep the same syntax for language filter as in other places? i.e.

```
:::note
* lang: csharp
<...>
:::
```

Reply:
I mimicked the syntax for code samples:

```markdown
```csharp
```
```

because I believe notes like this won't be cross-used between languages & this keeps it simpler. Do you think there's a genuine need for supporting multiple languages at the same time?